src/blockrelay/graphene_set.cpp

```diff
@@ -25,6 +29,11 @@ CGrapheneSet::CGrapheneSet(size_t _nReceiverUniverseItems,
     // Below is the parameter "m" from the graphene paper
     nReceiverUniverseItems = _nReceiverUniverseItems;
 
+    shorttxidk0 = _shorttxidk0;
+    shorttxidk1 = _shorttxidk1;
+    ibltSalt = ibltEntropy % CIblt::MaxNHash();
```

Comment:
is MaxNHash guaranteed to be != 0?

Reply:
The way it's currently setup in iblt.cpp, the smallest it can be is 4. But I redefined ibltSalt so that the divisor in the modulus is no less than 1, just to be safe.